RevenueCatUI/Templates/TemplateViewType.swift

```diff
@@ -148,7 +148,7 @@ extension PaywallData {
             // In case the offering was not updated yet, keep reading the previous selection value.
             let oldSelectionFallback: LinPaywall = offering.getMetadataValue(for: "show_new_paywall", default: false) ? .canvaStyleOneStep : .defaultRC
             let paywallVersion = LinPaywall(rawValue: offering.getMetadataValue(for: "paywall_version", default: oldSelectionFallback.rawValue)) ?? oldSelectionFallback
-            let step1Configuration: LinTemplateStep1Configuration = offering.getMetadataValue(for: "step1_paywall", default: .default)
+            let step1Configuration: LinTemplateStep1Configuration = offering.getMetadataValue(for: "step1_paywall") ?? .default
```

Comment:
I'm puzzled, isn't it the same?

Reply:
The new one uses Decodable. It is a little bit confusing API :D 
<img width="640" alt="Screenshot 2024-10-22 at 10 11 54" src="https://github.com/user-attachments/assets/10c1e3a8-62aa-4d8c-bfab-207186a4d99b">